doc/concept/granular-certificates/example.md

```diff
@@ -1,6 +1,6 @@
 # Example flow
 
-Below is an example wher two GCs are issued
+Below is an example where two GCs are issued
 
 - Consumption of 150Wh
 - Production of 250Wh
```

Comment:
Something I don't understand here: 

Does the consumption initiate production? l. 8: "The production is then sliced" - what happens before? Is this just to illustrate that consumption and production _can_ be matched? Not clear to me

I know this is an example, but some context might be helpful on how it is expected to work - What is the "matching" mechanism? In one of the other doc pages, it is desribed that only an equal amount of consumption and production slice can be matched, but I haven't found further info.

 Is there an automatic "slice" to make consumption and production equally large?
 What about "residual" slices? What if there are a looot of tiny slices in the end, can they be collected back into a slice?